contents/security/sqlInjection.md

```diff
@@ -0,0 +1,149 @@
+# SQL Injection #
+
+Author: [Lewis Koh](https://github.com/nus-cs3281/2018/blob/master/students/lewisKoh/lewisKoh-Resume.md)
+
+## Overview ##
+
+SQL (Structured Query Language) Injection is a vulnerability found in websites using databases.
+
+SQL is a common language which is used by websites to communicate with databases. Databases 
+can be used to store persistent data, such as usernames and passwords, sensitive account data, 
+or other important information used by the website. Databases are usually made of many "tables" 
+which are organised in rows and columns. Each row is a separate entry in the table, and 
+each column is a specific parameter which can be used by the entry.
```

Comment:
I think you can give an example of a table here, like in your talk. To connect it with the later part, maybe a Users table with simple username and password.

Reply:
I added a table. Not sure if it's what you were expecting, let me know! =)